### PR TITLE
fixed karate tests for bind-piece feature

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
https://folio-org.atlassian.net/browse/FAT-14355

## Approach
fixed failing circulation policy and bind-piece feature and made some improvements

### CODE REVIEW

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/bind-piece.feature [hunk 1]
```diff
@@ -55,12 +55,11 @@ Feature: Verify Bind Piece feature
     * call createOrder { id: '#(orderId)' }
 
     # 3. Create patron and user
-    * def userGroup = 'GROUP0369';
     * def v = call createUserGroup { id: '#(patronId)' }
     * def v = call createUser { id: '#(userId)', patronId: '#(patronId)'}
 
     # 4. Setup Circulation Policy
-    * call createCirculationPolicy { id: '#(policyId)', policyName: '#(policyName)' }
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/bind-piece.feature:63]: if we do not use policyId, policyName - we can remove these parameters from feature file

File: acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-circulation-policy.feature [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
   @CreateLoanPolicy
   Scenario: Create loan policy
-
     Given path 'loan-policy-storage/loan-policies'
     And request
       """
       {
-        "id": "#(id)",
-        "name": "#(name)",
+        "id": "d9cd0bed-1b49-4b5e-a7bd-064b8d177231",
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-circulation-policy.feature:12]: maybe using random uuid is more suitable here?

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/bind-piece.feature (hunk 1 above)
Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/bind-piece.feature:63]: thanks, removed.

File: acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-circulation-policy.feature (hunk 2 above)
Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-circulation-policy.feature:13]: added explanation in the beginning of the file. All @Policy ids is being used in @CirculationRules. If policy id has changes, appopriate id in rules need to be changed.

**Review by Saba-Zedginidze-EPAM — APPROVED** [review]

**Comment by azizbekxm** [comment]
image

**Pull request merged**